File: server/java/src/main/java/com/onlinepayments/example/service/domain/common/enums/Language.java
package com.onlinepayments.example.service.domain.common.enums;

public enum Language {
    English("en"),
    German("de"),
    French("fr");

    private final String sdkValue;

    Language(String sdkValue) {
        this.sdkValue = sdkValue;
    }

    public String getSdkValue() {
        return sdkValue;
    }
}

File: server/python/app/application/domain/enums/common/language.py
from enum import Enum

class Language(str, Enum):
    English = "English"
    French = "French"
    German = "German"

    @classmethod
    def has_value(cls, value: str) -> bool:
        return value in cls._value2member_map_

    def to_locale(self) -> str:
        locale_map = {
            Language.English: "en",
            Language.French: "fr",
            Language.German: "de",
        }
        return locale_map[self]